controller/main.go

```diff
@@ -124,7 +124,7 @@ func main() {
 		logLevel            = flag.String("log-level", "info", fmt.Sprintf("log level. must be one of: [%s]", logging.Levels.String()))
 		disableEpSlices     = flag.Bool("disable-epslices", false, "Disable the usage of EndpointSlices and default to Endpoints instead of relying on the autodiscovery mechanism")
 		enablePprof         = flag.Bool("enable-pprof", false, "Enable pprof profiling")
-		enableWebhook       = flag.Bool("enable-webhook", false, "Enable validation webhook")
+		enableWebhook       = flag.Bool("enable-webhook", true, "Enable validation and conversion webhooks")
```

Comment:
How about getting rid of the flag completely and always turn the webhooks on? Is there a reason why we wouldn't want that?
Even if we had to disable them because of some mis-behaviour, we could remove the webhook configuration.

Reply:
ok, sounds good to me.